.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,23 @@
+name: Github Pages
```

Comment:
Is the name freeform? I feel like "Deploy Storybook" should be reflected somewhere, like both

* filename
* `name` attribute

Reply:
Updated